cmd/dcrtime/dcrtime.go

```diff
@@ -936,6 +938,112 @@ func showWalletBalanceV2() error {
 	return nil
 }
 
+// lastAnchorV1 returns the last anchor information
+// such as: tx, chain timestamp, block height & block hash
+func lastAnchorV1() error {
+	c := newClient(*skipVerify)
+	route := *host + v1.LastAnchorRoute
+
+	fmt.Println(route)
```

Comment:
extraneous println?

Reply:
Intended, apparently this how we log in dcrtime.go